database/migrations/2020_06_23_190539_add_wmu_column_to_plots_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddWmuColumnToPlotsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('plots', function (Blueprint $table) {
+            $table->string('wmu', 2)->nullable();
```

Comment:
Indexing the column makes searching and filtering faster.

```suggestion
            $table->string('wmu', 2)->index()->nullable();
```